packages/ui/src/ui/pages/navigation/content/MapNode/MapNode.js

```diff
@@ -148,11 +151,14 @@ class MapNode extends Component {
 }
 
 function mapStateToProps(state) {
+    const path = getPath(state);
+
     return {
+        path,
+        showACOCreateButton: path === '//sys/access_control_object_namespaces/queries',
```

Comment:
It is better to check type of currrent path:
```ts
const type = getType(state);
return {
  allowCreateACO: type === 'access_control_object_namespace',
  ...
}
```

Reply:
We need to create an ACO only in queries space. Will this work correctly?